### DISCUSSION TITLE
Viewing and editing external data

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#15832] [Q&A]
What would be the best way to view and edit extrernal data in OrchardCore? I have watch the video [Orchard Harvest 2017 Outside In: Using external data with Orchard](https://www.youtube.com/watch?v=eRCwqnqBXzY ) is that still the best way to do it in OrchardCore as well? 

Basically I would like to add a module to OrchardCore to wrap around a (micro)service I wrote myself (comparable to the catalog data in the youtube video) which I would like to use as an external content type and content list. It would be nice to edit some properties of the data witch would be send to the service to do the actual update. I would like to use OrchardCore Cms to edit the UI for the end user and add 'normal' CMS items. 

Any help/pointers would be appreciated!!

**Comment by sebastienros** [comment]
Not sure if they can be Content Items, be you can provide them in the UI and add a custom controller to display them the same way we do for content items.
Or you would need to do imports, which could be triggered by the Workflow module or any custom controller, or background task.
